docs/project-setup.md

```diff
@@ -0,0 +1,78 @@
+# Setting up a new project
```

Comment:
Pretty sure "Project" is not the right term. "SIG" or "Language SIG" might be more accurate. Any preference?

Reply:
I'd go with SIG

Reply:
+1. I would say lets just call it a SIG

Reply:
sounds good!